src/components/Content.tsx

```diff
@@ -100,7 +100,7 @@ const Content: ContentComponent = props => {
     publish: publishMode(),
   }), [tutorialStep, isTutorialLocal])
 
-  return <div id='content-wrapper' onClick={e => {
+  return <div id='content-wrapper' data-testid='content-wrapper-test' onClick={e => {
```

Comment:
Could you help me understand the purpose of this a bit more? I see how it's being used but I don't totally understand the problem that it solves. What happens when only `screen` is used, and why?